examples/meta/src/meta_api/enums.sg

```diff
@@ -0,0 +1 @@
+enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC
```

Comment:
Java and C# don't allow statements that have no side effects like this one. 

One solution could be to try putting it in a print statement `print enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC`. The print statement doesn't work for arbitrary expressions because of C++, but I think it should work for something as simple as this. You may have to modify some of the `example/meta/generator/targets/*.json` files to get the this working (I think C++ needs an extra import statement for basic printing to work).

The other option is to find the simplest/most stable class in Shogun that has a method that accepts enums. The downside to this is that it couples the test with whatever class you choose which is not super clean.